examples/read_bids_datasets.py

```diff
@@ -0,0 +1,93 @@
+"""
+=========================================
+07. Read and load datasets in BIDS format
```

Comment:
I think this should be 01. or 02., because it's a very introductory example. The examples are ordered by difficulty.

Reply:
+1